openedx/core/djangoapps/programs/__init__.py

```diff
@@ -8,3 +8,10 @@
 To ensure maximum separation of concerns, and a minimum of interdependencies,
 this package should be kept small, thin, and stateless.
 """
+from openedx.core.djangoapps.waffle_utils import (WaffleSwitch, WaffleSwitchNamespace)
+
+PROGRAMS_WAFFLE_SWITCH_NAMESPACE = WaffleSwitchNamespace(name='programs_utils')
```

Comment:
I think @robrap 's nitpick comment on this was not only for the constant name but for the namespace itself (i.e. **name='programs_utils'** should be **name='programs'**)